hermes-api/src/main/java/pl/allegro/tech/hermes/api/MetricDecimalValue.java

```diff
@@ -0,0 +1,68 @@
+package pl.allegro.tech.hermes.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.Objects;
+
+import static java.lang.Double.parseDouble;
+
+public class MetricDecimalValue {
+    private static final String UNAVAILABLE_STRING = "unavailable";
+    private static final MetricDecimalValue UNAVAILABLE = new MetricDecimalValue(true, "-1.0");
+
+    private final boolean unavailable;
+    private final String value;
+
+    private MetricDecimalValue(boolean unavailable, String value) {
+        this.unavailable = unavailable;
+        this.value = value;
+    }
+
+    public static MetricDecimalValue unavailable() {
```

Comment:
Why we cannot make `UNAVAILABLE` as a public field and just use it?

Reply:
In my opinion `unavailable()` is better in this case because it is consistent with other factory method - `of(...)`.